### PR TITLE
adapt rating button styling

### PR DESCRIPTION
**Pull request by vellip**

### CODE REVIEW

File: meinberlin/assets/scss/components/_rating.scss [hunk 1]
```diff
@@ -39,12 +41,20 @@
         color: contrast-color($color);
     }
 
+    .fa {
+        margin-right: 0.3em;
+    }
```
Comment (xi) [inline meinberlin/assets/scss/components/_rating.scss]: In other places we use the `i` element for styling.

File: meinberlin/assets/scss/components/_rating.scss [hunk 2]
```diff
@@ -39,12 +41,20 @@
...
     &.is-read-only,
     &:disabled {
         color: $color;
         background: none;
         border: 0;
     }
+
+    .comment & {
+        padding: 0.3em 0.6em;
+    }
```
Comment (xi) [inline meinberlin/assets/scss/components/_rating.scss]: Why is this necessary. Can we use a modifier instead?

Comment (vellip) [inline meinberlin/assets/scss/components/_rating.scss]: Ratings are render via react and ratings for comments are supposed to be smaller by design. So even if we'd use a modifier we'd have to alter the core.

Comment (xi) [inline meinberlin/assets/scss/components/_rating.scss]: Ok, can we move this code to `comment.scss` or `shame.scss` then?

**Comment by xi** [comment]
I was wondering whether `.rating-button` should share code with `.button`. @damusp said no.
